airbyte-integrations/connectors/source-instagram/components.py

```diff
@@ -67,6 +66,15 @@ def get_http_response(name: str, path: str, request_params: Dict, config: Config
     return response.json()
 
 
+def remove_params_from_url(url, params):
```

Comment:
this used to be in common.py, but moved this here since we need a unified `components.py` file